src/main/java/de/tum/in/www1/artemis/web/rest/ComplaintResource.java

```diff
@@ -149,10 +151,30 @@ public ComplaintResource(AuthorizationCheckService authCheckService, ExerciseSer
         return ResponseEntity.ok(responseComplaints);
     }
 
+    /**
+     * Get /complaints/for-tutor-dashboard/:exerciseId
+     * <p>
+     * Get all the complaints associated to an exercise, but filter out the ones that are about the tutor who is doing the request, since tutors cannot act on their own complaint
+     *
+     * @param exerciseId the id of the exercise we are interested in
+     * @return the ResponseEntity with status 200 (OK) and a list of complaints. The list can be empty
+     */
+    @GetMapping("/complaints/for-tutor-dashboard/moreFeedback/{exerciseId}")
```

Comment:
The URL does not make much sense. Please refactor it and use a proper URL scheme.
(If similar URL schemes are used at the moment in this resource, please refactor them as well)

Reply:
I changed it and updated documentation. I don't have to change URL schemes in other places, so I don't want to move just this single method to a new resource. 
Maybe it would be better to have a single complaints endpoint and return two lists: complaints and more feedback requests. But I think it is better to separate them